From 24f587ab0128c688f38ebf0241bc431a79860b7c Mon Sep 17 00:00:00 2001
From: Chandni Verma <chandni@linux.vnet.ibm.com>
Date: Mon, 7 Nov 2016 15:58:41 +0530
Subject: [PATCH 30/43] Fix typo "classes" when checking if temp sysfs got
 mounted

When I ran a test to mount sysfs in a temp. dir, I found that
no "/classes" sub-dir exists. Only "/class" was found on Linux.

Signed-off-by: Chandni Verma <chandni@linux.vnet.ibm.com>
Signed-off-by: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
---
 src/core/sysfs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/sysfs.cc b/src/core/sysfs.cc
index 2b09554..97dbab5 100644
--- a/src/core/sysfs.cc
+++ b/src/core/sysfs.cc
@@ -58,7 +58,7 @@ struct sysfs_t
           NULL);
       }
 
-      has_sysfs = exists(path + "/classes/.");
+      has_sysfs = exists(path + "/class/.");
     }
   }
 
-- 
2.10.2

